Content.Shared/Remotes/DoorRemoteComponent.cs

```diff
@@ -0,0 +1,21 @@
+using Robust.Shared.Analyzers;
+using Robust.Shared.GameObjects;
+
+namespace Content.Shared.Remotes
+{
+    [RegisterComponent]
+    [Friend(typeof(DoorRemoteSystem))]
+    public class DoorRemoteComponent : Component
+    {
+        public override string Name => "DoorRemote";
+
+        public OperatingMode Mode = OperatingMode.OpenClose;
+
+        public enum OperatingMode
```

Comment:
```suggestion
        public enum OperatingMode : byte
```

Would also be nice to have an IItemStatus for it on the client